udata_gouvfr/theme/gouvfr/__init__.py

```diff
@@ -46,8 +46,6 @@
     nav.Item(_('Data'), 'datasets.list'),
     nav.Item(_('Reuses'), 'reuses.list'),
     nav.Item(_('Organizations'), 'organizations.list'),
-    nav.Item(_('Dashboard'), 'site.dashboard'),
-    nav.Item(_('Documentation'), None, url='https://doc.data.gouv.fr'),
```

Comment:
Documentation unused link ? 😢

Reply:
This will be addressed in etalab/data.gouv.fr#228